src/authentication/steps/SelectServer.jsx

```diff
@@ -0,0 +1,162 @@
+/* global __ALLOW_HTTP__ */
```

Comment:
Almost the same file as we already had, allows the user to enter the URL of the cozy. In this version, most of the URL validation happens inside the component and not in a redux action.